lib/tasks/import/log.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+desc 'Import daily log from Access CSV at data/daily_log.csv'
+namespace :import do
+  task log: :environment do
+    CSV.foreach('data/daily_log.csv', encoding: 'ISO-8859-1', headers: true).with_index 2 do |row, line|
+      date = Date.parse row.fetch('Date:')
+      user_name = row.fetch('Dispatcher:')
+      if user_name.is_a? String
+        last_name, first_name = user_name.split(', ')
+        user = User.find_by name: [first_name, last_name].join(' ')
```

Comment:
I can see that it won't throw an error if user is nil, so that's okay.

I think it may be important to know who submitted prior log entries - would it make sense to just add a line that asks if user is nil, add the username like `" - submitted by #{user_name}" at the end of the note?